src/Helper/WebformHelper.php

```diff
@@ -42,21 +42,22 @@ public function webformThirdPartySettingsFormAlter(array &$form, FormStateInterf
   /**
    * Define the email templates to use for sending emails through this module.
    *
-   * @param $key
+   * @param string $key
    *   The mail key.
-   * @param $message
+   * @param array $message
    *   The message to send.
-   * @param $params
+   * @param array $params
    *   Various parameters used by the mail template.
    *
-   * @return void
+   * @phpstan-param array<string, mixed> $message
+   * @phpstan-param array<string, mixed> $params
    */
-  public function mail($key, &$message, $params) {
+  public function mail(string $key, array &$message, array $params): void {
     if ($key == 'submission_log_notification') {
       $message['from'] = $params['from'];
       $message['subject'] = $params['title'];
       $message['body'][] = (new FormattableMarkup($params['message'], []))->__toString();
```

Comment:
Is `(new FormattableMarkup($params['message'], []))->__toString()` the same value as `$params['message']`?

Reply:
I believe the 2nd parameter allows for some placeholders.

Reply:
Yes, but we don't use any placeholders here, so the final result must be the same as `$params['message']` unless I'm missing something.